src/test/scala/centaur/StandardTestCaseSpec.scala

```diff
@@ -27,9 +27,7 @@ class StandardTestCaseSpec extends FlatSpec with Matchers with ParallelTestExecu
   // Optional test cases are provided by the end user as opposed to the ones built in to the system
   val optionalTestCases = CentaurConfig.optionalTestPath map testCases getOrElse List.empty
 
-  optionalTestCases ++ testCases(CentaurConfig.standardTestCasePath) foreach {
-    t => executeStandardTest(t, t.testFunction)
-  }
+  optionalTestCases ++ testCases(CentaurConfig.standardTestCasePath) foreach { t => executeStandardTest(t, t.testFunction) }
```

Comment:
that does look like a bug waiting to happen (except I know it isn't, since it's been working for a while, but ...)